charts/argo-cd/templates/argocd-configs/cluster-secrets.yaml

```
@@ -22,7 +22,7 @@ stringData:
   {{- if $cluster_value.shard }}
   shard: {{ $cluster_value.shard | quote }}
   {{- end }}
-  name: {{ required "A valid .Values.configs.clusterCredentials.CLUSTERNAME.name entry is required!" $cluster_key }}
+  name: {{ default $cluster_key $cluster_value.name }}
```

```suggestion
  name: {{ $cluster_key }}
```

The simplest solution I see is to drop the required function